mlir/lib/Dialect/Linalg/Transforms/TransposeMatmul.cpp

```diff
@@ -31,6 +31,12 @@ using namespace mlir::linalg;
 FailureOr<Operation *> mlir::linalg::transposeMatmul(RewriterBase &rewriter,
                                                      linalg::MatmulOp matmulOp,
                                                      bool transposeLHS) {
+  // Check to not let go the matmul with extended semantic, through this
+  // transform.
+  if (matmulOp.hasUserDefinedMaps())
```

Comment:
nit: for multiple lines it would be still good to add `{`...`}`

Reply:
Added.